src/ServicePulse.Host/app/js/views/archive/view.html

```diff
@@ -51,7 +51,7 @@
                     </div>
 
                     <div class="col-xs-11 failed-message-data" ng-mouseenter="message.hover2 = true" ng-mouseleave="message.hover2 = false">
-                        <a href="#/failedMessages/{{message.id}}">
+                        <a href="#/message/{{message.id}}">
```

Comment:
From a nav perspective, `/failedMessages` is parent of both groups and individual messages, so I'd nest both children under the parent. I'd also avoid two words in oneWord to improve legibility of the URL, as well as caps. So I'd use this structure in the end:

- `/failed-messages`
- `/failed-messages/group/{{group.id}}`
- `/failed-messages/message/{{message.id}}`